embedded/store/immustore.go

```diff
@@ -851,7 +851,7 @@ func (s *ImmuStore) syncBinaryLinking() error {
 	}
 	defer s.releaseAllocTx(tx)
 
-	txReader, err := s.newTxReader(s.aht.Size()+1, false, true, tx)
+	txReader, err := s.newTxReader(s.aht.Size()+1, false, true, false, tx)
```

Comment:
I feel we should either declare these true/false as named conditional variables, so that readability and understanding is simple because it is getting complicated reading it
```
DisablePrecommittedTx = false
EnablePrecommittedTx = true

s.newTxReader(s.aht.Size()+1, DisablePrecommittedTx, DisableIntegrityCheck, DisableY, tx)
```

Reply:
I'm totally agree. I think the way to go would be to have optional parameters as it's done in the sdk